instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/TemporaryMetricsView.java

```diff
@@ -61,10 +63,10 @@ private static Set<AttributeKey> buildActiveRequestsView() {
     // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attributes
     Set<AttributeKey> view = new HashSet<>();
     view.add(SemanticAttributes.HTTP_METHOD);
-    view.add(SemanticAttributes.HTTP_HOST);
     view.add(SemanticAttributes.HTTP_SCHEME);
     view.add(SemanticAttributes.HTTP_FLAVOR);
-    view.add(SemanticAttributes.HTTP_SERVER_NAME);
+    view.add(SemanticAttributes.NET_HOST_NAME);
+    // TODO: net host port?
```

Comment:
Opened https://github.com/open-telemetry/opentelemetry-specification/issues/2822 for this